demos/jans-tarp/src/options/oidcClients.tsx

```diff
@@ -88,11 +88,16 @@ function Row(props: { row: ReturnType<typeof createData> }) {
                 chrome.storage.local.set({ oidcClients: clientArr.filter(obj => obj.clientId !== row.clientId) });
             }
         });
+        handleNotifyOnDataChange();
     }
 
+    const handleNotifyOnDataChange = () => {
```

Comment:
This function is unnecessary, you can directly use **notifyOnDataChange** function directly.